src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_CalendarView.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Private.Infrastructure;
+using Windows.UI.Xaml.Controls;
+using MUXControlsTestApp.Utilities;
+using Windows.UI.Xaml.Media;
+using Uno.UI.RuntimeTests.MUX.Helpers;
+using Windows.UI;
+using Windows.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Controls;
+using System.Reflection;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls;
+
+#if __WASM__
```

Comment:
Any reason the test only runs on Wasm?

Reply:
Because I pushed without any condition but azure build was failed. So I am going to check if this fix pass the azure test.

Reply:
I tested this Given_CalendarView.cs in SamplesApp.Wasm and passed but I don't know why azure test failed.
@Youssef1313 Could you help me this problem?

Reply:
The current failure of Pipeline Validations macOS is unrelated to your changes. I'll restart the CI once it finishes.

Reply:
So there isn't need any preprocessors?

Reply:
We will see depending on the final outcome of the CI :)

Reply:
Ok, I will remove all preprocessors.